nix/wrapper.nix

```
@@ -1,8 +1,8 @@
 { ixxPkgs, lib, nuscht-search, pkgs }:
 
 rec {
-  mkOptionsJSON = pkgs.callPackage ({ modules, specialArgs, nixosOptionsDoc }: (nixosOptionsDoc {
-    inherit ((lib.evalModules {
+  mkOptionsJSON = pkgs.callPackage ({ modules, specialArgs, nixosOptionsDoc, overrideEvalModulesArgs }: (nixosOptionsDoc {
```

```suggestion
  mkOptionsJSON = pkgs.callPackage ({ modules, specialArgs, nixosOptionsDoc, overrideEvalModulesArgs ? { } }: (nixosOptionsDoc {
```
We expose this in the flake.nix but I am not sure why to be honest. If @MarcelCoding has no strong opinion on removing it, I would like to preserve some backwards compatibility.

---

Keeping this exposed seems reasonable to me, as it provides the functionality to generate the options json of multiple modules. But I would like to include Sandro's change to keep backward compatibility.